docs/getting-started/smr-emulation.md

```diff
@@ -164,16 +164,21 @@ Zone 00079: type 0x2 (Sequential-write-required), cond 0x1 (Empty), reset recomm
 ## *scsi_debug*
 
 The *scsi_debug* kernel module can be used to create emulated ZBC SCSI disks
-with memory backing used to store data written to sectors. Since memory is used
-as a backing store, creating large disks requires a host with a large amount of
-DRAM. Furthermore, the data written to the emulated device does not survive the
-device destruction or a host reboot.
+that use memory backing used to store data, which is written to sectors.
+Because this method uses memory as a backing store, the creation of large disks
+requires a host with a large amount of DRAM. 
+
+!!! Note 
+
+    This method creates volatile memory. This means that the data written to
```

Comment:
may creates volatile memory -> uses volatile memory, since we are not creating DRAM :)
Will fix when applying